src/components/pages/home/index.tsx

```diff
@@ -11,15 +12,35 @@ import Link from 'next/link'
 import {track} from 'utils/analytics'
 import Jumbotron from 'components/pages/home/jumbotron'
 import {Jumbotron as HolidayReleaseJumbotron} from 'components/pages/20-days-of-egghead/course-grid'
+import toast, {Toaster} from 'react-hot-toast'
 
 const Home: React.FC<any> = ({data, jumbotron, location}) => {
+  const router = useRouter()
+
+  React.useEffect(() => {
+    const {query} = router
+    if (query.message) {
+      toast(query.message as string, {
+        icon: '✅',
+        duration: 5000,
+      })
+    }
+  }, [router])
+
+  React.useEffect(() => {
+    const {query} = router
+    if (query.type === 'claimed') {
+      router.reload()
+    }
+  }, [])
```

Comment:
When a gift is claimed, we set a `type` in the url to `claimed` which will force a reload for the user so they can see the content they now have access to